plugins/mcreator-core/procedures/world_send_chat.json

```diff
@@ -5,24 +5,67 @@
       "name": "text",
       "check": "String"
     },
+    {
+      "type": "field_dropdown",
```

Comment:
Since you can technically define any color, I have added new color chooser field to the core: https://github.com/MCreator/MCreator/commit/52bbb9380314679d4e633e9f15f15a7dfa8cdc6d

Please adapt this block to use it and merge master branch in your PR so it will work in your PR